src/js/views/eva-variant-view.js

```diff
@@ -591,6 +592,33 @@ EvaVariantView.prototype = {
         this.allAssociatedSSIDs = Array();
     },
 
+    setSpeciesAndAssemblyAccession: function(accessionID) {
+        var accessionNumber = this.accessionID.substring(2);
+        var category = this.accessionID.startsWith("rs") ? "clustered-variants": "submitted-variants";
+
+        var accessioningServiceData;
+        EvaManager.get({
+            host:EVA_ACCESSIONING_HOST,
+            version: EVA_VERSION,
+            category: category,
+            resource: accessionNumber,
+            async: false,
+            success: function (response) {
+                try {
+                    console.log(response);
+                    accessioningServiceData = response;
+                } catch (e) {
+                    console.log(e);
+                }
+            }
+        });
+        var data = accessioningServiceData[0].data;
+        this.assemblyAccession = this.accessionID.startsWith("rs") ? data.assemblyAccession : data.referenceSequenceAccession;
+
+        var speciesInfo = _.findWhere(getEVASpeciesList(), {assemblyAccession : this.assemblyAccession});
+        this.species = speciesInfo.taxonomyCode + "_" + speciesInfo.assemblyCode;
```

Comment:
I would prefer if these variable names were not called species since it contains a mixture of species and assembly.
I thin they refer the the database name.

Reply:
Agreed offline not to change the name as `species` is widely used in the website. In a future refactor it can be modified to `dataseName`